### PR TITLE
add verbose flag to cli to

### PR DESCRIPTION
**Pull request by Madhu-1**
log error message to stdout

fix for #629  
Signed-off-by: Madhu Rajanna <mrajanna@redhat.com>

### CODE REVIEW

File: glustercli/cmd/volume-set.go [hunk 1]
```diff
@@ -31,6 +31,7 @@ func init() {
 	volumeSetCmd.Flags().BoolVar(&flagSetAdv, "advanced", false, "Allow setting advanced options")
 	volumeSetCmd.Flags().BoolVar(&flagSetExp, "experimental", false, "Allow setting experimental options")
 	volumeSetCmd.Flags().BoolVar(&flagSetDep, "deprecated", false, "Allow setting deprecated options")
+	volumeSetCmd.Flags().BoolVarP(&verbose, "verbose", "v", false, "verbose output")
```
Comment (prashanthpai) [inline glustercli/cmd/volume-set.go]: This is done at a lot of places. Cobra has provision to set a flag on the root command and all subcommands will inherit that flag.

See `glustercli/cmd/root.go`:

```go
RootCmd.PersistentFlags().BoolVarP(...
```

File: glustercli/cmd/root.go [hunk 2]
```diff
@@ -13,7 +13,8 @@ var RootCmd = &cobra.Command{
 	Use:   "glustercli",
 	Short: "Gluster Console Manager (command line utility)",
 	PersistentPreRun: func(cmd *cobra.Command, args []string) {
-		err := logging.Init(flagLogDir, flagLogFile, flagLogLevel, false)
+		//err := logging.Init(flagLogDir, flagLogFile, flagLogLevel, false)
```
Comment (prashanthpai) [inline glustercli/cmd/root.go]: Remove the comment.

File: glustercli/cmd/root.go [hunk 3]
```diff
@@ -13,7 +13,8 @@ var RootCmd = &cobra.Command{
...
+		err := logging.Init("", "stdout", flagLogLevel, false)
```
Comment (prashanthpai) [inline glustercli/cmd/root.go]: We don't have to use `logging.Init()`. If we use logrus directly, we can customize the formatter too,

**Comment by Madhu-1** [comment]
@prashanthpai  fixed review comments

File: glustercli/cmd/root.go (hunk 3 above)
Comment (Madhu-1) [reply glustercli/cmd/root.go]: I was intentionally left it,  wanted to make use of current function

**Pull request merged**